src/app/main/component/events/components/event-editor/components/create-event-dates/components/date-time/date-time.component.ts

```diff
@@ -1,5 +1,5 @@
 import { AfterViewInit, Component, ElementRef, Input, OnInit, ViewChild } from '@angular/core';
-import { AbstractControl, FormArray, FormGroup } from '@angular/forms';
+import { AbstractControl, FormArray, FormControl, FormGroup, Validators } from '@angular/forms';
 import { DateAdapter, MAT_DATE_FORMATS, MAT_DATE_LOCALE } from '@angular/material/core';
 import { IMask } from 'angular-imask';
 import * as _moment from 'moment';
```

Comment:
Unused imports specifier `Validators` and `FormControl`

Import can be shortened: './moment-date-adapter/moment-date-adapter'